server/bin/index-pbench

```diff
@@ -362,12 +281,10 @@ class ResultData(PbenchData):
         Fetch the list of result.json files for this experiment;
         return a list of dicts containing their metadata.
         """
-        paths = [x for x in ptb.tb.getnames() if x.find("result.json") >= 0 and ptb.tb.getmember(x).isfile()]
+        paths = [x for x in ptb.tb.getnames() if (os.path.basename(x) == "result.json") and ptb.tb.getmember(x).isfile()]
         datafiles = []
         for p in paths:
-            fname = os.path.basename(p)
-            datafile = _dict_const(path=p, basename=fname)
-            datafiles.append(datafile)
+            datafiles.append(os.path.dirname(p))
```

Comment:
So this is a list of directories where a "result.json" is found? Should the name of the function be changed?

Reply:
Fixed.